src/values/instruction_value.rs

```diff
@@ -113,6 +115,14 @@ impl<'ctx> InstructionValue<'ctx> {
     fn is_a_alloca_inst(self) -> bool {
         !unsafe { LLVMIsAAllocaInst(self.as_value_ref()) }.is_null()
     }
+    #[llvm_versions(10.0..=latest)]
+    fn is_a_atomicrmw_inst(self) -> bool {
+        !unsafe { LLVMIsAAtomicRMWInst(self.as_value_ref()) }.is_null()
+    }
+    #[llvm_versions(10.0..=latest)]
+    fn is_a_cmpxchg_inst(self) -> bool {
```

Comment:
Can you add a test for this? I think you should be able to slide it into the existing tests somewhere

Reply:
Done. It gets a new test function so that I can make it llvm10+ only.